src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -294,87 +295,85 @@ protected internal override void AddResponseFileCommands(CommandLineBuilderExten
         /// list of aliases, and if any of the aliases specified is the string "global",
         /// then we add that reference to the command-line without an alias.
         /// </summary>
-        private void AddReferencesToCommandLine
-            (
-            CommandLineBuilderExtension commandLine
-            )
+        internal static void AddReferencesToCommandLine(CommandLineBuilderExtension commandLine, ITaskItem[] references, bool isInteractive = false)
         {
             // If there were no references passed in, don't add any /reference: switches
             // on the command-line.
-            if ((this.References == null) || (this.References.Length == 0))
```

Comment:
I missed that one, I fixed up most of the other instance to just check for null and let the enumerator handle the empty case.